000-vip-init.php

```diff
@@ -12,6 +12,8 @@
 use Automattic\VIP\Utils\Context;
 use Automattic\VIP\Utils\WPComVIP_Restrictions;
 
+// @codeCoverageIgnoreStart
```

Comment:
Can we add a comment noting why we're doing this?

Reply:
@sjinks I believe you added this line. I would say it is related with the SonarCloud report, but I wanted to double check and ask for your input before adding a comment here.

Reply:
@mjangda please disregard this for now - I will write a P2 post explaining all these things.

In brief, PHPUnit is unable to generate a coverage report for the files loaded from the bootstrap script, even if those files are used in tests (I suspect that the code needs to be instrumented when it is compiled, and compilation of the bootstrap script and the files it requires happens before the code coverage driver is initialized). This is why these files are reported as uncovered.

I want to try another approach: initialize the autoloader and load the test library in `bootstrap.php` and use PHPUnit's Test Listener feature to load all mu-plugins before the very first test (the driver should be initialized by that time). However, I am not sure this will play nice with the way the WordPress Test Library initializes all hooks and filters.